src/Scaffolding/VS.Web.CG.EFCore/EntityFrameworkModelProcessor.cs

```diff
@@ -554,10 +556,62 @@ private DbContext TryCreateContextUsingAppCode(Type dbContextType, Type startupT
             }
             catch (Exception ex)
             {
+                System.Diagnostics.Debugger.Launch();
+                var exceptionType = ex.GetType();
+                // if MySQL exception with error code 1045, discard error message since it contains sensitive dev information
+                if (exceptionType.Name.Equals(MySqlException, StringComparison.OrdinalIgnoreCase) || exceptionType.FullName.Contains(MySqlException, StringComparison.OrdinalIgnoreCase))
```

Comment:
Is there a reason to do something other than a full string comparison against  exceptionType.FullName?

Reply:
The exception comes from the MySQL provider and there are plenty of options out there. I tried two different providers and based on that, a FullName check seem to satisfy both of them (they were the two most popular).